app/views/how-tos/make-first-prototype/let-user-change-answers.html

```diff
@@ -0,0 +1,67 @@
+{% set pageTitle = "Let the user change their answers" %}
+{% set next = { "title" : "Show different pages depending on user input" , "link" : "branching" } %}
+{% set prev = { "title" : "Show the user’s answers on your ‘Check answers’ page" , "link" : "show-users-answers" } %}
+
+{% extends 'how-tos/make-first-prototype/_BASE.njk' %}
+
+{% block makePrototype %}
+  <h2 id="make-the-change-links-work">Make the ‘Change’ links work</h2>
+<p>Make the <strong>Change</strong> links on the ‘{{examples[3].title}}’ page work by adding the right links.</p>
+<ol>
+<li>In the <code>&lt;a&gt;</code> tag under <code>{{ data['how-many-balls'] }}</code>, change the href attribute from <code>#</code> to <code>/{{examples[1].url}}</code></li>
+<li>In the <code>&lt;a&gt;</code> tag under <code>{{ data['most-impressive-trick'] }}</code>, change the href attribute from <code>#</code> to <code>/{{examples[2].url}}</code></li>
+</ol>
+<p>If you select a <strong>Change</strong> link, you’ll go back to the right question page, but your answer will not appear yet.</p>
+<h2 id="show-the-users-answer-in-question-1">Show the user’s answer in question 1</h2>
+<p>Radios and checkboxes have a <code>checked</code> option to set whether they are selected (checked) or not when the page loads.</p>
+<p>Open the <code>{{examples[1].url}}.html</code> file in your <code>app/views</code> folder.</p>
+<p>For each of the <code>items</code>, we’ll add a <code>checked</code> value, like this:</p>
+<pre tabindex="0"><code>    
+
+&#123;
+  value: "yes",
+  text: "Yes",
+  checked: checked("magical-powers", "yes")
```

Comment:
Doh, I didn’t know we even had this filter in the NHS prototype kit... I've been using `checked: data['magical-powers'] === 'yes'`.

At some point we should also add copy GOV.UK and let you set `value:` on the top-level macro option, to save having to set `checked` on each individual item, as that’d be far simpler.